Core/Graphics/Source/Android/Graphics.cpp

```diff
@@ -0,0 +1,15 @@
+#include <Babylon/Graphics.h>
+#include "../GraphicsImpl.h"
+#include <GraphicsPlatform.h>
+
+#include <AndroidExtensions/Globals.h>
+#include <AndroidExtensions/JavaWrappers.h>
+
+namespace Babylon
+{
+    float Graphics::Impl::GetDevicePixelRatio()
+    {
+        auto dpi = android::global::GetAppContext().getResources().getConfiguration().getDensityDpi();
+        return (float)dpi/160.0f;
```

Comment:
What is this division by 160? Will you add a comment and a link to associated Android documentation?

Reply:
160 dpi is the "baseline" android pixel density:
https://developer.android.com/training/multiscreen/screendensities

I'm currently looking to see if there are any constants to reference instead of using a magic number